abandon notice (#412)
symfony-cmf/cmf-sandbox: README.md

# This repository is no longer maintained

Due to lack of interest, we had to decide to discontinue this repository. 

> The CMF project will focus on the [Routing component](https://github.com/symfony-cmf/routing) and [RoutingBundle](https://github.com/symfony-cmf/routing-bundle) which is the only part that is actively used.

Reviewer: ```suggestion
The CMF project focusses on the [Routing component](https://github.com/symfony-cmf/routing) and [RoutingBundle](https://github.com/symfony-cmf/routing-bundle), which are still in active use by other projects.
```

> This repository will no longer be upgraded and marked as abandoned, but will be kept available for legacy projects or if somebody wants to experiment with the CMF.

Author: ```suggestion
This repository will no longer be upgraded and marked as abandoned, but will be kept available for legacy projects or if somebody wants to experiment with the CMF.

You can contact us in the #symfony_cmf channel of the [Symfony devs slack](https://symfony.com/slack).
```
